File: src/main/java/net/dashflight/data/mfa/BasicUserData.java
package net.dashflight.data.mfa;

import java.util.UUID;
import lombok.Builder;
import lombok.Data;

@Data
public class BasicUserData {

    /**
     * The user's unique id
     */
    private UUID userId;

    /**
     * The unique secret associated with the user
     */
    private String userSecret;

    /**
     * The email address of the user
     */
    private String email;

}

File: src/main/java/net/dashflight/data/mfa/BasicUserData.kt
package net.dashflight.data.mfa

import java.util.*


data class BasicUserData(var userId: UUID? = null, var userSecret: String? = null, var email: String? = null) {
    /**
     * The user's unique id
     */

    /**
     * The unique secret associated with the user
     */

    /**
     * The email address of the user
     */
}